core/src/main/java/org/apache/mahout/common/distance/EuclideanDistanceMeasure.java

```diff
@@ -29,26 +31,28 @@
  * squared differences.
  */
 public class EuclideanDistanceMeasure extends SquaredEuclideanDistanceMeasure {
-  
+  Logger log = LoggerFactory.getLogger(EuclideanDistanceMeasure.class);
   @Override
   public double distance(Vector v1, Vector v2) {
     double largeDistanceValue = 1000000;
-    int [] fieldNumbersThatMustMatch = {3}; // Field 3 is dwellingType
+    int [] fieldNumbersThatMustMatch = {6}; // Field 3 is dwellingType
```

Comment:
@sjayrams @kxfeldhus is this correct , is it 3 or is it 6 ?  The custom build was done with 3 or 6 ?